biapol_utilities/transform/_warp_polar_5D.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+
+from skimage.transform import warp_polar
+import numpy as np
+
+
+def warp_polar_5D(image, center=None, radius=None, project=None,
```

Comment:
Not sure if I completely understand it: The function wraps the polar transformation and applies it to image stacks along c and t? If that is the case, I think I would *tend* to remove the projection function from this function. It makes the function rather complex and can be easily used on the output of the function. This could be nicely explained in a notebook!

Reply:
Yes, since we are talking about polar/cylindric transformation, the coordinate system changes only happen in (x,y) to (radius, angle). The idea was essentially to put that in a single function to avoid writing code to iterate over other dimensions everytime and also to return maximum projections with a single letter parameter instead of, for example, specifying `axis=0` or `axis=(0,2)`.

Taking that out will simplify the function code, I agree, projections would need to be done manually afterwards then.

Reply:
I am a bit in favor of removing the projection function here because I would intuitively expect that a transform function just transforms stuff. 

What a find a bit confusing from this point of view, is that the transform function returns exclusively the projection if the project setting. But since it is `None` by default I think it may as well stay. @haesleinhuepf, do you have an opinion on this?

Reply:
The transformation always happens, because it goes from cartesian to polar, but a projection can or cannot be returned depending on the parameter `project`. That is why I put `None` by default, because the projection is optional.

I am considering taking the projection out of the function now, because, the way it is, it assumes a certain axes order. Furthermore, I will try to implement it in a different way.

Reply:
A bit further down, the function returns with 

```Python
return(np.amax(image_polar, axis=str2dim[project]))
```
in case `project is not None`

I think in this case the function ends with returning only the projection, but not the transformation.

Reply:
Ah, I see, you mean the expected behaviour in this case should be to return both, is that it?

Reply:
That's what I mean, I don't really have a good answer to this question ^^" @haesleinhuepf @lazigu do you have an opinion on this?

Reply:
I tend to write many simple functions that do only one thing each, and then I combine them. For me this is easier to read:

```
projection( transformation( image ))
```

than this:

```
transformation( image, projection=True)
```

Furthermore, this is not possible with solution 1:

```
projection( image )
```

The underlying code is also easier to maintain, because it is "modular".  [See also](https://en.wikipedia.org/wiki/Separation_of_concerns)

Reply:
Ok makes sense, I am working to generalize the function and I will leave the projection out of it.

Reply:
The function warp_polar_5D was deleted. Instead, I created a more general function to apply a function to images whose dimensions are bigger than the function accepted input. One would usually need to loop through these dimensions, but this functions does that using dask with lazy calls to the function.